lib/happo/public/HappoDiffs.jsx

```diff
@@ -56,11 +69,34 @@ function renderNewImages(newImages) {
         </a>
       </h2>
 
-      {newImages.map(renderImage)}
+      {newImages.map(renderNewImage)}
     </div>
   );
 }
 
+const HappoDiff = React.createClass({
+  propTypes: {
+    image: PropTypes.shape(imageObjectStructure),
+  },
+
+  render() {
+    const {
+      image,
+    } = this.props;
+
+    return (
+      <div>
+        {renderImageHeading(image)}
+        <div className='happo-diff__images'>
```

Comment:
nit: What do you think about PascalCasing the block on these to match the component? `HappoDiff__images`